.github/workflows/slack-pr-notifications.yaml

```
@@ -0,0 +1,754 @@
+
+          if [ "$HTTP_CODE" = "200" ]; then
+            echo "✅ Successfully sent Slack notification (HTTP $HTTP_CODE)"
+          else
+            echo "⚠️ Slack webhook returned HTTP $HTTP_CODE"
+            echo "Response: $RESPONSE"
+            # Don't fail the workflow if Slack notification fails
+          fi
+
+      - name: Add PR comment (for opened/reopened)
+        if: github.event.action == 'opened' || github.event.action == 'reopened'
+        uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)